etcd-recovery/etcdrecovery.go

```diff
@@ -66,8 +66,12 @@ func NewRecoveryCommand() *cobra.Command {
 	cmd.PersistentFlags().StringVar(&opts.etcdClientKeyFile, "etcd-client-key", opts.etcdClientKeyFile, "etcd client cert key file.")
 	cmd.PersistentFlags().StringVar(&opts.etcdCAFile, "etcd-ca-cert", opts.etcdCAFile, "etcd trusted CA cert file.")
 	cmd.PersistentFlags().StringVar(&opts.namespace, "namespace", "", "namespace of etcd cluster")
-	_ = cmd.MarkFlagRequired("etcd-endpoints")
-	_ = cmd.MarkFlagRequired("namespace")
+
+	err := cmd.MarkPersistentFlagRequired("namespace")
```

Comment:
It's necessary to check the err and os.Exit? I mean, it will automatically do it exiting with `os.Exit(1)` and giving this error msg: `Error: required flag(s) "namespace" not set`

Reply:
Yea it's necessary as this err returned is more of a "misconfiguration" error that needs to be caught at runtime. Like marking a flag required that doesn't exist. So in our case this will happen if later one someone added a code that changed the `namespace` flag name without changing this line to the new name.